core/src/modules/ops/setupSteps.ts

```diff
@@ -53,6 +54,19 @@ export namespace SetupStepOps {
         return setting.value !== setting.defaultValue;
       },
     },
+    add_a_model: {
+      key: "add_a_model",
+      title: "Add a Model",
+      description:
+        "Models define the shape of your Records.  Grouparoo uses a Profile Model by default.  You can add new Models and customize them when you begin adding Sources and Properties.",
```

Comment:
We may want to tweak this language a bit, but this is 👍 for now!

Reply:
Happy to tweak it if you have a suggestion!

Reply:
Should I merge it now?  The docs links go to a 404 currently as the models/records updates haven't been done yet.